test/uk/gov/hmrc/nisp/utils/BreadcumbSpec.scala

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.nisp.utils
+
+import org.scalatestplus.play.OneAppPerSuite
+import play.api.test.FakeRequest
+import uk.gov.hmrc.nisp.config.ApplicationConfig
+import uk.gov.hmrc.nisp.controllers.auth.NispUser
+import uk.gov.hmrc.play.frontend.auth.connectors.domain.{PayeAccount, Accounts, ConfidenceLevel, CredentialStrength}
+import uk.gov.hmrc.play.frontend.auth.{Principal, LoggedInUser, AuthContext}
+import uk.gov.hmrc.play.test.UnitSpec
+
+class BreadcrumbSpec extends UnitSpec with OneAppPerSuite {
```

Comment:
Will the tests compile without `ÒneAppPerSuite`? It will make them run faster if there is no play application